packages/common/src/components/PkgGraph.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+  ReactWidget
+} from '@jupyterlab/apputils';
+
+import { Graph, GraphData, GraphNode, GraphLink } from "react-d3-graph";
+
+import * as React from 'react';
+
+import { Conda } from '../tokens';
+
+export class PkgGraph extends ReactWidget {
```

Comment:
Could you refractor this into a thin `ReactWidget` and a `React` component?

The React component would have:
- props
  - pkgManager
  - package
- state
  - data: {nodes, links} | null

Call `_updatePackages` in `componentDidMount` and `componentDidUpdate`

`_loading` is unneeded and could be replaced with `_data === null`

The idea is to have a component that will be easily extendable to display the dependency graph of an environment.